api/src/main/java/com/exelaration/abstractmemery/controllers/MemeController.java

```diff
@@ -12,18 +11,19 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import com.exelaration.abstractmemery.domains.Caption;
-
 @RestController
 @RequestMapping("/meme")
 @CrossOrigin(origins = "http://localhost:3000")
 public class MemeController {
 
-    @PostMapping("/")
-    public ResponseEntity addMeme(@RequestBody Caption caption) {
-        Logger logger = LoggerFactory.getLogger(UploadController.class);
-        logger.info("\nPost Request\nTop Text: " + caption.getTopText() + "\nBottom Text: " + caption.getBottomText());
-        return ResponseEntity.ok(HttpStatus.OK);
-    }
-
-}
\ No newline at end of file
+  @PostMapping("/")
+  public ResponseEntity addMeme(@RequestBody Caption caption) {
+    Logger logger = LoggerFactory.getLogger(UploadController.class);
```

Comment:
This should be `MemeController.class`.  Should we just remove the logging altogether here?